apps/redi-talent-pool/src/services/api/api.tsx

```diff
@@ -349,6 +353,35 @@ export async function fetchAllTpJobListings(): Promise<Array<TpJobListing>> {
   return resp.data.filter((listing) => !listing.dummy)
 }
 
+export async function fetchExpiredTpJobListings(): Promise<
+  Array<TpJobListing>
+> {
+  const userId = getAccessTokenFromLocalStorage().userId
+  const currentDate = new Date()
+  const resp = await http(
+    `${API_URL}/redUsers/${userId}/tpJobListings?filter=${JSON.stringify({
+      where: {
+        expiresAt: { lt: currentDate },
+      },
```

Comment:
I am testing this PR on my local and see neither active nor expired job postings. Not sure if the problem is here, but it seems that fetching job listings returns undefined.